Flickr/Sources/FlickrAppVCs/SearchTabDelegates.swift

```diff
@@ -10,3 +10,7 @@ import Foundation
 protocol SearchScreenViewControllerDelegate: AnyObject {
     func didTapSearchButton(searchString: String)
 }
+
+protocol SearchResultsViewControllerDelegate: AnyObject {
+    func didSelectImage(url: URL, title: String, imageTitle: String)
```

Comment:
Please declare these delegates in the same file as ViewCOntrollers and Coordinators.